utils/scripts/find_duplicate_read_sets/find_duplicate_read_sets.py

```diff
@@ -0,0 +1,118 @@
+
+        # iterate by page within a read set
+        for page in paginator.paginate(sequenceStoreId=seq_store):
+
+            # get just the read set information
+            for rs in page['readSets']:
+                read_set_id = rs['id']
+                if rs.get('etag',''):
+
+                    # there may be 2 sources if the file type is a FASTQ
+                    etag_source_1 = rs['etag'].get('source1','')
+                    etag_source_2 = rs['etag'].get('source2','')
+                    file_type = rs.get('fileType')
+                    name = rs.get('name')
+
+                    # only log if a etag was found
+                    if etag_source_1:
+                        s1 = {
+                            'sequence_store_id': seq_store,
+                            'read_set_id': read_set_id, 
+                            'name': name,
+                            'type': file_type,
+                            'source': 'source 1', 
+                            'etag': etag_source_1
+                        } 
+                        df = pd.concat([df, pd.DataFrame([s1])])
+                    if etag_source_2:
+                        s2 = {
+                            'sequence_store_id': seq_store,
+                            'read_set_id': read_set_id, 
+                            'name': name,
+                            'type': file_type,
+                            'source': 'source 2', 
+                            'etag': etag_source_2
+                        } 
+                        df = pd.concat([df, pd.DataFrame([s2])])
+
+    # filters to only duplicates based on the etag field
+    filtered_df = df[df.duplicated(subset='etag', keep=False)]
+
+    # sorts for easy identification
+    df_sorted = filtered_df.sort_values(by=['etag','read_set_id'], ascending=True)
+
+    return df_sorted
+
+
+
+# Driver Code
+
+# Decide the two file paths according to your
+# computer system
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('-s', '--seq_store_ids', type=str, required=True, dest='seqStoreIds', 
+                        help='A comma separated list of 1 or more sequence store IDs to check, all in the same region.')
+    parser.add_argument('-r', '--region', type=str, required=True, dest='region', 
+                        help='The region where the sequence stores are in. They must all be in the same region.')
+    parser.add_argument('-o', '--output', type=str, default='duplicate_read_sets.csv', dest='output', 
+                        help='(optional) The path and file name for the output csv file containing all the duplicate read sets. Include the file \
+                        name and prefix (e.g. "~/path/to/out/duplicates.csv"). The default file name is duplicate_read_sets.csv and will write to \
+                        the current directory.')
+    parser.add_argument('-p', '--profile-name', type=str, default='NA', dest='profileName', 
```

Comment:
omitting `default=` from `add_argument` will set the default value to `None`. This simplifies passing this optional argument when creating a `boto3.Session`.

Reply:
done